### PR TITLE
Enable `reactions-avatars` on more pages

### LINKED ISSUE #3978: Enable `reactions-avatars` on discussions  and releases
Such as https://github.com/tophf/mpiv/discussions/50

### PR DESCRIPTION
**Pull request by kidonng**
Fix #3978

## Test URLs

- Discussion: https://github.com/tophf/mpiv/discussions/50
- Release: https://github.com/sindresorhus/refined-github/releases/tag/21.9.3

## Screenshot

image

image

### CODE REVIEW

File: source/features/reactions-avatars.tsx [hunk 1]
```diff
@@ -1,5 +1,7 @@
 import './reactions-avatars.css';
 import React from 'dom-chef';
+import onetime from 'onetime';
+import {observe} from 'selector-observer';
 import select from 'select-dom';
 import {flatZip} from 'flat-zip';
 import * as pageDetect from 'github-url-detection';
```
Comment (yakov116) [inline source/features/reactions-avatars.tsx:7]: Sort

**Comment by fregante** [comment]
The problem is that hasComments automatically uses its own listener, you can see it in features/index.tsx

It's probably best to create 2/3 inits. One for hasComments, one with the observer for the discussions, one static for the releases

**Comment by kidonng** [comment]
> The problem is that hasComments automatically uses its own listener, you can see it in features/index.tsx

I forgot that

> It's probably best to create 2/3 inits. One for hasComments, one with the observer for the discussions, one static for the releases

That's what I did initially, I just thought this is simpler

File: source/features/reactions-avatars.tsx (hunk 1 above)
Comment (kidonng) [reply source/features/reactions-avatars.tsx]: Is there any guidelines for this other than looking at other files? I know some of them but it's not clear.

**Pull request merged**